### PR TITLE
build.md: emulator building instructions for NixOS

### PR DESCRIPTION
**Pull request by mmahut**

### CODE REVIEW

File: core/docs/build.md [hunk 1]
```diff
@@ -58,7 +64,7 @@ make build_unix
 Now you can start the emulator:
 
 ```sh
-./emu.sh
+bash ./emu.sh
```
Comment (prusnak) [inline core/docs/build.md]: Why is this required? On my NixOS system this works fine.

File: core/docs/build.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ Install the required packages, depending on your operating system.
   sudo pacman -S scons sdl2 sdl2_image
   ```
 
+* __NixOS__:
+
+  ```sh
+  nix-shell -p gnumake python3 scons SDL2_image SDL2.dev pkg-config
```
Comment (prusnak) [inline core/docs/build.md]: There is `shell.nix` file in the root of the project. I usually run `nix-shell` in the root (which picks up `shell.nix`) and then `cd core` to do core stuff.

File: core/docs/build.md (hunk 1 above)
Comment (mmahut) [reply core/docs/build.md]: Even without using patchShebangs? Interesting, I just get bad interpreter.

File: core/docs/build.md (hunk 2 above)
Comment (mmahut) [reply core/docs/build.md]: That's probably the right way to do it for the developement. I will rebase.

File: core/docs/build.md (hunk 1 above)
Comment (prusnak) [reply core/docs/build.md]: I wonder whether we need bash actually. Maybe it would be better to get rid of any bashisms for the script and change the shebang to `#/bin/sh`. What do you think?

Comment (prusnak) [reply core/docs/build.md]: Removed bashisms from emu.sh in 917f5c6534e5089c8864513b86cabdc2b402f742

**Comment by prusnak** [comment]
Applied the more complex change in b89d1db0e49d1dc001e0a6808862c46d75e3f1f9

Thanks!
